addon.py

```
@@ -25,49 +27,117 @@
             self.stop()
-            
+
     def stop(self):
         self.running = False
-        
+
         # Close socket
         if self.socket:
             try:
                 self.socket.close()
             except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)